packages/addons/service/tvheadend42/source/defaults/config

```diff
@@ -1,13 +1,14 @@
 {
-	"uilevel": 1,
-	"uilevel_nochange": false,
+	"wizard": "hello",
+	"uilevel": 0,
+	"uilevel_nochange": true,
 	"ui_quicktips": true,
 	"cookie_expires": 14,
 	"caclient_ui": true,
 	"epg_compress": true,
 	"prefer_picon": true,
 	"chiconpath": "file:///storage/picons/tvh/%C.png",
 	"chiconscheme": 2,
-	"piconpath": "file:///storage/picons/vdr/",
+	"piconpath": "file:///storage/picons/tvh/",
 	"piconscheme": 1
```

Comment:
This wasn't planed this way, at addon dev we "found" this nice bug we are not able to probably work around (only 2-3 file/sec get extracted regardless of size and storage speed). Atm no proper solution at the horizon and for now just remain it as we have it. Or do you have an better idea ?